### PR TITLE
[MUL-306] extend draft with tax category

### PR DESCRIPTION
**Pull request by Kriz83**

### CODE REVIEW

File: translations/messages.en.yml [hunk 1]
```diff
@@ -166,6 +166,7 @@ open_marketplace:
         vendor_test_credentials: Vendor test credentials
         username: Username
         password: Password
+        tax_category: Tax Category
```
Comment (senghe) [inline translations/messages.en.yml:169]:
```suggestion
        tax_category: Tax category
```

File: features/vendor/create_product_listing_with_tax_category.feature [hunk 2]
```diff
@@ -0,0 +1,96 @@
+@vendor_managing_product_listings
```
Comment (senghe) [inline features/vendor/create_product_listing_with_tax_category.feature:1]: We discussed on the call to refactor the scenarios

**Review by senghe — CHANGES_REQUESTED** [review]

File: tests/Behat/Context/Setup/ProductListingContext.php [hunk 3]
```diff
@@ -12,103 +12,205 @@
...
+    public function thereIsARejectedProductListingCreatedByVendor(): void
     {
         $vendor = $this->sharedStorage->get('vendor');
+        $productListing = $this->createProductListing(
+            $vendor,
+            DraftInterface::STATUS_REJECTED
+        );
+
+        $this->entityManager->persist($productListing);
+//        $this->entityManager->flush();
```
Comment (senghe) [inline tests/Behat/Context/Setup/ProductListingContext.php:108]:
```suggestion
```

File: tests/Behat/Context/Setup/ProductListingContext.php [hunk 4]
```diff
@@ -123,4 +225,121 @@ public function iShouldBeNotifiedNoPageExits()
         $status = $this->getSession()->getStatusCode();
         Assert::eq($status, 404);
     }
+
+    /**
+     * @Then I fill in conversation message content with :message
+     */
+    public function iFillInConversationMessageContentWithMessage($message): void
+    {
+        $this->getSession()
```
Comment (senghe) [inline tests/Behat/Context/Setup/ProductListingContext.php:234]: We should create separate page class for this (please see the `test/Behat/Page` directory)

File: tests/Behat/Context/Ui/Vendor/ProductListingContext.php [hunk 5]
```diff
@@ -136,6 +136,16 @@ public function iShouldSeeProductsListingStatus($status)
         Assert::notNull($productListingStatus);
     }
 
+    /**
+     * @Then I should see :count product listing(s)
+     */
+    public function iShouldSeeProductListings($count)
+    {
+        $rows = $this->getPage()->findAll('css', 'table > tbody > tr');
```
Comment (senghe) [inline tests/Behat/Context/Ui/Vendor/ProductListingContext.php:144]: We should use separate page class for this.

**Review by senghe — CHANGES_REQUESTED** [review]

File: features/vendor/create_product_listing_with_tax_category.feature (hunk 2 above)
Comment (Kriz83) [reply features/vendor/create_product_listing_with_tax_category.feature:1]: done

**Review by senghe — APPROVED** [review]

**Pull request merged**